.github/workflows/standards-and-tests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         os: [ubuntu-20.04]
         epubcheck: [4.2.4]
         prince: [14.2-1]
-        wordpress: [5.9.3, latest]
+        wordpress: ['6.0', latest]
```

Comment:
have to use quotes for now (see https://github.com/actions/runner/issues/849 for explanation of why)